contracts/BerfChatStorage.sol

```diff
@@ -36,7 +35,9 @@ contract BerfChatStorage {
     /// in the same order into the hashing function,
     /// to generate the exact same chat id hash.
     /// @param _to Address of the message recipient.
-    function sendMessage(address _to/*, bytes32 _messageHash*/) public {
+    /// @param _messageHash String that is the hash
+    /// of the message contents (generated off-chain)
+    function sendMessage(address _to, string memory _messageHash) public {
```

Comment:
usually better to leave `msg.sender` as this is more explicit

Reply:
Does it cost the same amount of gas (or less) to read the transaction property of `msg.sender` than if we were to assign it to a local variable and then reading that local variable? That was the reason behind this. 

I know that reading from storage is more expensive than reading from memory (`SLOAD` vs `MLOAD` opcodes), and I thought I read that reading from memory was cheaper, gas wise, than reading the tx data.

Will research further to confirm whether that is correct or not.